moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -657,7 +657,7 @@ const ProjectFundingTable = () => {
                   ...newData,
                   project_id: projectId,
                   added_by: getDatabaseId(user),
-                  funding_status_id: 1,
+                  funding_status_id: newData.funding_status_id || 1,
```

Comment:
This works great with the unspecified default being tentative or user input if provided. 🙌